### PR TITLE
Sidebar improvements

### PR DESCRIPTION
**Pull request by motechFR**
Hide access to settings section if user is not a space admin

Add in a link to the Charmverse discord for support

image

### CODE REVIEW

File: components/common/CreateSpaceForm.tsx [hunk 1]
```diff
@@ -75,6 +75,7 @@ export default function WorkspaceSettings ({ defaultValues, onSubmit: _onSubmit,
         spaceRoles: {
           create: [{
             role: 'admin',
+            isAdmin: true,
```
Comment (mattcasey) [inline components/common/CreateSpaceForm.tsx:78]: nice catch

File: components/common/PageLayout/components/Sidebar.tsx [hunk 2]
```diff
@@ -167,6 +168,8 @@ export default function Sidebar ({ closeSidebar, favorites }: SidebarProps) {
   const [isScrolled, setIsScrolled] = useState(false);
   const favoritePageIds = favorites.map(f => f.pageId);
 
+  const isAdmin = user?.spaceRoles.some(role => role.spaceId === space?.id && (role.isAdmin === true || role.role === 'admin'));
```
Comment (mattcasey) [inline components/common/PageLayout/components/Sidebar.tsx:171]: can we deprecate/delete the use of "role" now? why is it still a valid way to be admin?

Comment (Devorein) [reply components/common/PageLayout/components/Sidebar.tsx:171]: I have the same question. Why do we have two ways to check if someone is an admin 😕

**Comment by motechFR** [comment]
Closed as #424 handled this in further detail

**Pull request closed without merging**